src/flow.ts

```diff
@@ -228,8 +221,24 @@ export class Flow {
    * @returns an array of Font objects corresponding to the provided `fontNames`.
    */
   static setMusicFont(...fontNames: string[]): Font[] {
+    // HACK during review to introduce the correct stacks step by step
+    switch (fontNames[0]) {
+      case 'Bravura':
+        CommonMetrics.fontFamily = 'Bravura,BravuraText';
+        break;
+      case 'Gonville':
+        CommonMetrics.fontFamily = 'GonvilleSmufl,Bravura,Roboto Slab';
```

Comment:
This is interesting and unexpected for me. The font stack includes two music fonts and one text font.

I guess that sort of makes sense... if our ChordSymbol is a text string that includes a mix of music symbols and ASCII text symbols?

Reply:
It is indeed expected. The approach in Vexflow5 is to specify all the fonts and the browser will fall back to the next one when the glyph is not defined. We search in SMuFL first and then in standard text. My fault was to believe that BravuraText was a standard text file to combine with Bravura.